.circleci/config.yml

```diff
@@ -140,7 +140,7 @@ jobs:
                   ENV_VARS_STRING=$(echo "$ENV_VARS" | jq -r '.environmentVariables')
 
                   # Add 'export' to each line and write to file
-                  echo "$ENV_VARS_STRING" | sed 's/^/export /' > ~/project/environment_variables.txt
```

Comment:
why remove the seed export replace?

Reply:
also let's call this .env.ef_circleci just in case?

Reply:
cause it will put export on each line of the .env file and we don't need it